lib/transformations/index.js

```diff
@@ -38,7 +38,8 @@ function transform(source, transforms, options) {
 	}, options);
 	transforms = transforms || Object.keys(transformations).map(k => transformations[k]);
 	transforms.forEach(f => f(jscodeshift, ast));
-	return ast.toSource(recastOptions);
+	const astStr = ast.toSource(recastOptions);
+	return prettier.format(astStr, {singleQuote: true, tabWidth: 2});
```

Comment:
Users should be able to pass prettier options. Also, some options like `singleQuote` and `tabWidth` might overlap recast options. We need to respect both or remove recast options completely.